<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOP Design Studio - SECS/GEM Script Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Alarm">
       <attribute name="title">
        <string>ALARM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_condition">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_bracketOpen">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>{</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_line1">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_line2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_bracketClose">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_AlarmId">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Alarm ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_AlarmId">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_AlarmCount">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_AlarmCount">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Device Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditDevAddr">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>[@PLC1:D%1.%2]</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollAreaDevAddr">
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>659</width>
                 <height>371</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>6</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButtonAddDevParam">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButtonDelDevParam">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="tableWidgetDevParams">
                  <property name="textElideMode">
                   <enum>Qt::ElideMiddle</enum>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Default</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Increase</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Increase
Per</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Base</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Field
Width</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Use
Range</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Min</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Max</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Virtual Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditVirAddr">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>[@Virtual:Alarm_%1:1:UDEC]</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollAreaVirAddr">
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>659</width>
                 <height>371</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButtonAddVirParam">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButtonDelVirParam">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="tableWidgetVirParams">
                  <property name="textElideMode">
                   <enum>Qt::ElideMiddle</enum>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Default</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Increase</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Increase
Per</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Base</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Field
Width</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Use
Range</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Min</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Max</string>
                   </property>
                   <property name="textAlignment">
                    <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_GenAlarm">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
